thorlcr/activities/funnel/thfunnelslave.cpp

```diff
@@ -376,7 +382,11 @@ class FunnelSlaveActivity : public CSlaveActivity
 
             auto startInputNFunc = [&](unsigned i)
             {
-                try { startInput(i); }
+                try
+                {
+                    LookAheadTimer timer(slaveTimerStats, timeActivities);
```

Comment:
should not include for i == 0, because it is not called async/on a separate, see deferred comment below